### ISSUE TITLE
Experimental: Lots of text is not readable due to too low contrast between foreground and background

### ISSUE THREAD

**Issue by fladd** [vector-im/element-web#7946]
* **Left pane:** Rooms with new messages are not distinguishable from rooms with no new messages
* **Middle pane:**
  * **Top:** Room description is not readable
  * **Center:** Dates, timestamps, join/leave messages are not readable
  * **Bottom:** Encryption status, message in the entry field are not readable
* **Right pane:**
  * **Member list:** Online/offline state, message in filter box are not readable
  * **File list:** Timestamps, filesize are not readable
  * **Notifications:** Dates are not readable

**Comment by jryans** [comment]
@fladd, could you include a screenshot as well? Some users have been seeing different fonts than expected, so it's good to have visual confirmation of what you are seeing on your side.

**Comment by RRREEEEEEEE** [comment]
What theme? Can't reproduce this on any of them, I can read all the text.

**Comment by jryans** [comment]
> What theme? Can't reproduce this on any of them, I can read all the text.

Since this is a redesign issue, I am assuming the reporter is talking about the theme on riot.im/experimental. (There is only one theme currently available there.)

**Comment by lampholder** [comment]
WCAG contrast checker (set to AA) flags issues with:
- hyperlinks
- timestamps
- room topic text
- _some_ of the display name colours
- the 'Invite to this room' white on green button in the member list panel

Everything else is fine (explicitly: timeline text, member list, room list, avatars, room title).

It looks to me like the ff WCAG plugin doesn't check for visibility of placeholder text, 'cause the placeholder in the room- and memberlist-filters looks _very_ low contrast to me.

**Comment by AndrewJDR** [comment]
Particularly the "Online/offline state" not being visible in the member list anymore is an annoyance. It's a pretty common thing to glance over at that status to know whether it's worth contacting someone at the moment.

**Comment by nadonomy** [comment]
Closing this as >2 years old, themes have changed dramatically since. :)

If needed, please open new issue(s) with screenshots.
